docs/source/api/user_current.rst

```diff
@@ -169,28 +169,6 @@ Request Structure
 
 Response Structure
 ------------------
-:addressLine1:     The user's address - including street name and number
```

Comment:
@ocket8888 I removed this to match the documentation for `user/current/update POST`. The response example is below and doesn't return the actual user information. Do you know if this response is something we want to include in the future for this endpoint?

Reply:
Yes. All `PUT` requests should print out the resulting object representation in the response.
Did you modify `PUT /api/1.x/user_current` such that it no longer does that?

Reply:
I did not modify it. That is how it is. The response example shows the current output.

Reply:
The `PUT /api/1.x/user/current` endpoint is not written in go yet.

Reply:
okay

Reply:
If we're going to be consistent with API 2.0, `POST`s and `PUT`s should probably all return the same data that can be retrieved by doing a subsequent `GET`. _Most_ of our Perl endpoints did that, but nearly all of the golang-rewritten versions fail to do that currently, which is technically a breaking-API regression.

Anyways, thanks for cleaning up this lying documentation. 👍